rules/kube_apiserver-config.libsonnet

```diff
@@ -8,9 +8,9 @@
     // These are buckets that exist on the apiserver_request_sli_duration_seconds_bucket histogram.
     // They are what the Kubernetes SIG Scalability is using to measure availability of Kubernetes clusters.
     // If you want to change these, make sure the "le" buckets exist on the histogram!
-    kubeApiserverReadResourceLatency: '1',
-    kubeApiserverReadNamespaceLatency: '5',
-    kubeApiserverReadClusterLatency: '30',
-    kubeApiserverWriteLatency: '1',
+    kubeApiserverReadResourceLatency: '1(\.0)?',
+    kubeApiserverReadNamespaceLatency: '5(\.0)?',
+    kubeApiserverReadClusterLatency: '30(\.0)?',
+    kubeApiserverWriteLatency: '1(\.0)?',
```

Comment:
```suggestion
    kubeApiserverReadResourceLatency: '1(.0)?',
    kubeApiserverReadNamespaceLatency: '5(.0)?',
    kubeApiserverReadClusterLatency: '30(.0)?',
    kubeApiserverWriteLatency: '1(.0)?',
```

Reply:
Of course, you're right @SuperQ. What you need instead then is to double-double-escape! i.e.

```diff
-    kubeApiserverReadResourceLatency: '1(\.0)?',
-    kubeApiserverReadNamespaceLatency: '5(\.0)?',
-    kubeApiserverReadClusterLatency: '30(\.0)?',
-    kubeApiserverWriteLatency: '1(\.0)?',
+    kubeApiserverReadResourceLatency: '1(\\\\.0)?',
+    kubeApiserverReadNamespaceLatency: '5(\\\\.0)?',
+    kubeApiserverReadClusterLatency: '30(\\\\.0)?',
+    kubeApiserverWriteLatency: '1(\\\\.0)?',
```

which would produce queries like:

```
sum by (cluster) (cluster_verb_scope_le:apiserver_request_sli_duration_seconds_bucket:increase30d{verb=~"POST|PUT|PATCH|DELETE",le=~"1(\\.0)?"})
```

Reply:
I've done the double-double-escape on the following PR instead:

- #1008